src/register/containers/register/register.component.html

```diff
@@ -9,6 +9,12 @@ <h1 class="govuk-heading-xl">Register to manage civil and family law cases</h1>
           <a href="#" [routerLink]="'/register-org/register/organisation-name'" role="button" class="govuk-button govuk-button--start">Start</a>
       </ng-container>
 
+      <div *ngIf="pageId === 'check' && (errorMessage | async) !== '' && !(isFromSubmitted$ | async)">
+        <app-notification-banner-component>
```

Comment:
create function to remove if

Reply:
Implemented like the rest of the checks in this component - not to sure if can be done in ts file (uses pipes for both values) ?